client/client_update_test.go

```diff
@@ -811,6 +811,36 @@ var waysToMessUpServer = []swizzleExpectations{
 		}},
 }
 
+var _waysToMessUpServerRoot []swizzleExpectations
```

Comment:
as `waysToMessUpServerRoot` returns this var, it seems like it should be declared inside that function?